benchmark_runner/oadp/oadp_workloads.py

```diff
@@ -2805,6 +2805,33 @@ def this_is_downstream(self):
         if self.__test_env['source'] != 'upstream':
             return True
 
+    @logger_time_stamp
+    def invoke_log_collection(self, scenario):
+        """
+        method invokes misc-scripts/benchmark-runner-wrapper/log_collector.sh
+        required to pass crname from tests.yaml
+        """
+
+        log_collector_sh = (f"{self.__oadp_misc_dir}/benchmark-runner-wrapper/log_collector.sh")
+        cr_name = scenario['args']['OADP_CR_NAME']
+        logger.info(f"invoke_log_collection is attempting to collect logs from cr: {cr_name} and write logs to dir: {self._run_artifacts_path}")
+        log_collection_output = self.__ssh.run(cmd=f"{log_collector_sh} {cr_name} --enable-collect-events --logs-folder {self._run_artifacts_path}")
+        logger.info(f"log_collection_output: {log_collection_output}")
+
+        logger.info(f"Getting the list of file names that exist in {self._run_artifacts_path} folder")
+        list_of_files = list(self.__ssh.run(cmd=f"find {self._run_artifacts_path} -type f").splitlines())
+        logger.info(f"{list_of_files}")
+
+        logger.info(f"Getting the size of the files in {self._run_artifacts_path} folder")
+        list_of_files_size = []
+        for filename in list_of_files:
+            list_of_files_size.append(self.__ssh.run(cmd=f"stat -c %s {filename}"))
+
+        logger.info(f"Check if file size is zero")
+        for i in range(len(list_of_files_size)):
+            if list_of_files_size[i] == "0":
+                logger.info(f"The file size of: {list_of_files[i]} is {list_of_files_size[i]} bytes")
```

Comment:
this is good, regarding the logging of the file size, what we might be worried about is logging an error if the file size is zero,  and if so maybe we throw a logger.error and the message of which file is zero ? WDYT?